ChangeLog

```diff
@@ -1,20 +1,26 @@
 Until present
 
-- Coding style set to LLVM style; all sources in src/ and include/
-  auto-formatted to that
-- Code modernized for compilation with C++ compilers; language changed to C99
-  and C++11 for all future development
 - 'pet' branch merged into master; pluto now needs pet and thus clang
   headers/libraries to build
+- Introduced loop unroll and jam support using ClooG AST and enable it by
+  default.
+- Improvements to intra-tile optimization heuristics.
+- Support for the LP based decoupled framework, Pluto-lp-dfp, included;
+  requires Pluto to be configured with GLPK or Gurobi.
+- Two more fusion models, namely, typed and hybrid fusion incorporated. These
+  models use the decoupled framework work.
+- libpluto interface updated.
+- --tile and --parallel enabled by default
+- Diamond tiling enabled by default
+- Check to identify time-iterated stencils with concurrent start is
```

Comment:
Drop "is implemented" - if it's in Changelog and in the pluto repo, it's obviously implemented! Also, isn't this stencil identification just for relevant for pluto-lp-dfp? Change to

- Auto identification of time-iterated stencils with concurrent start for pluto-lp-dfp

?